<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JNUEletricFee</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Sources/icon.ico</normaloff>Sources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="flushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
     </font>
    </property>
    <property name="text">
     <string>Flush</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dormitoryInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>宿舍号</string>
    </property>
   </widget>
   <widget class="QLabel" name="dormitoryNote">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>请按照Txx-xxxx的形式输入宿舍号</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>剩余费用:</string>
    </property>
   </widget>
   <widget class="QLabel" name="waterUsed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本月用水:</string>
    </property>
   </widget>
   <widget class="QLabel" name="electricUsed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本月用电:</string>
    </property>
   </widget>
   <widget class="QLabel" name="electricPrice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电费单价:</string>
    </property>
   </widget>
   <widget class="QLabel" name="waterPrice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>水费单价:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="balanceDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="waterDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="electricDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="electricFeeDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="waterPriceDisplay">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>   @copyright Wang Yuchen</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
